kutt.go

```diff
@@ -111,6 +111,12 @@ func (cli *Client) ReplaceLinks(ctx context.Context, target string) (error, stri
 	r := xurls.Strict()
 	raw := r.FindAllString(target, -1)
 	for _, r := range raw {
```

Comment:
Unrelated to this change but it would probably be nicer not to shadow the regex var. Maybe:
```suggestion
	matches := r.FindAllString(target, -1)
	for _, match := range matches {
```
Plus other renames etc.

Reply:
That makes a lot of sense. I'll put it together in a different PR in case this change is bad somehow.